ocs_ci/ocs/platform_nodes.py

```diff
@@ -475,6 +475,19 @@ def create_and_attach_volume(self, node, size, ssd=False):
         vm = self.get_vms([node])[0]
         self.vsphere.add_disk(vm=vm, size=size, ssd=ssd)
 
+    def create_and_attach_volumes(self, node, volume_sizes, ssd=False):
```

Comment:
this method doesn't exist as an abstract method in `NodesBase`. 
Same applies to `create_and_attach_volume`